fastlane/lib/fastlane/setup/crashlytics_beta.rb

```diff
@@ -61,7 +66,7 @@ def lane_template
     crashlytics(api_token: '#{@beta_info.api_key}',
              build_secret: '#{@beta_info.build_secret}',#{crashlytics_path_arg}
                    emails: ['#{@beta_info.emails.join("', '")}'], # You can list more emails here
-                 # groups: ['group_alias_1', 'group_alias_2'], # You can define groups on the web and reference them here
+                 # groups: ['#{groups.join("', '")}'], # You can define groups on the web and reference them here
```

Comment:
Yes, we're iterating on this feature, so we'll eventually want this not commented out but right now this is the desired behavior 👍